PowerEditor/src/WinControls/ProjectPanel/ProjectPanel.cpp

```diff
@@ -398,7 +398,8 @@ bool ProjectPanel::openWorkSpace(const TCHAR *projectFileName)
 
 	NativeLangSpeaker *pNativeSpeaker = (NppParameters::getInstance()).getNativeLangSpeaker();
 	generic_string workspace = pNativeSpeaker->getAttrNameStr(PM_WORKSPACEROOTNAME, "ProjectManager", "WorkspaceRootName");

```

Comment:
Doesn't your change make this line (400) obsolete?

Reply:
That's observed correctly. Thank you. I should remove it.
The translation itself is used now only in `ProjectPanel::newWorkSpace()`.